docs/source-2.0/guides/model-translations/converting-to-openapi.rst

```diff
@@ -691,6 +691,29 @@ onErrorStatusConflict (``String``)
             }
         }
 
+.. _generate-openapi-setting-syncCorsPreflightIntegration:
```

Comment:
This setting is API-Gateway specific, so should be moved down to [the API Gateway settings section](https://github.com/smithy-lang/smithy/blob/main/docs/source-2.0/guides/model-translations/converting-to-openapi.rst?plain=1#L1450). It's worth also adding a note to [the API Gateway specific CORS section](https://github.com/smithy-lang/smithy/blob/main/docs/source-2.0/guides/model-translations/converting-to-openapi.rst?plain=1#L1645).

Reply:
```suggestion
.. _generate-openapi-apigateway-setting-syncCorsPreflightIntegration:
```